.github/workflows/main.yml

```diff
@@ -25,13 +25,45 @@ jobs:
     - run: tox -e ${{ matrix.env }}
 
   pre-commit:
-    runs-on: ubuntu-latest
+    name: pre-commit
```

Comment:
needed in order to comply to our standards, since we're using local hook for mypy